Signal/src/environment/ExperienceUpgrades/ExperienceUpgrade.swift

```diff
@@ -54,7 +54,7 @@ class ExperienceUpgrade: TSYapDatabaseObject {
         // these models in a "change log" archive.
         if propertyKey == "title" || propertyKey == "body" || propertyKey == "image" {
             return MTLPropertyStorageNone
-        } else if propertyKey == "uniqueId" || propertyKey == "seenAt" {
+        } else if propertyKey == "uniqueId" || propertyKey == "seenAt" || propertyKey == "hasEverBeenSaved" {
```

Comment:
I finally added a `hasEverBeenSaved` property to `TSYapDatabaseObject`.  Let's discuss!